High fds (#205)
cvmfs/doc-cvmfs: apx-issues.rst

Known Issues
============

> Having a Very Large Number as File Descriptor Limit

Reviewer: ```suggestion
Slow Performance with a Very Large File Descriptor Limit
```
